worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -486,6 +528,14 @@ final private[worker] class StorageManager(conf: CelebornConf, workerSource: Abs
             case e: Exception => logWarning("Clean expired HDFS shuffle failed.", e)
           }
         }
+        if (db != null) {
+          committedFileInfos.remove(shuffleKey)
```

Comment:
Seems we should always call `committedFileInfos.remove(shuffleKey)` no matter db is null or not, because `notifyFileInfoCommitted` is always called.

Reply:
Then we'd better pass null as storageManager to FileWriter to avoid calling notify